src/ui/public/vislib/visualizations/point_series/seri_types.js

```diff
@@ -0,0 +1,12 @@
+import VislibVisualizationsColumnChartProvider from 'ui/vislib/visualizations/point_series/column_chart';
+import VislibVisualizationsLineChartProvider from 'ui/vislib/visualizations/point_series/line_chart';
+import VislibVisualizationsAreaChartProvider from 'ui/vislib/visualizations/point_series/area_chart';
+
+export default function SeriTypeFactory(Private) {
+
+  return {
+    histogram: Private(VislibVisualizationsColumnChartProvider),
```

Comment:
At some point (not now probably) we should rename this to be "bars" so that it matches the others. A histogram is more than just bars.